src/xunit.performance.api/ETWProfiler.cs

```diff
@@ -107,9 +116,142 @@ public static void Profile(string assemblyFileName, string sessionName, Action a
 
             TraceEventSession.MergeInPlace(userFullFileName, Console.Out);
             WriteInfoLine($"ETW Tracing Session saved to \"{userFullFileName}\"");
+            collectOutputFilesCallback(userFullFileName);
+
+            var assemblyModel = GetAssemblyModel(assemblyFileName, userFullFileName, sessionName, performanceTestMessages);
```

Comment:
Is the AssemblyModel an Xunit specific construct?  I don't think we want anything xunit specific here, given that we're going to have tests that don't execute under xunit.